CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```

Comment:
This is great, but we must be careful that it does not becomes the dictature of minorities against the majority. The minorities must feel welcome and equal, but the minorities should not gain special right because of their status. See a great example of how this could turn out very badly with the example of the Evergreen campus: https://www.youtube.com/watch?v=u54cAvqLRpA

Reply:
Agree